Gemfile.lock

```diff
@@ -128,6 +129,7 @@ GEM
       io-console (~> 0.5)
     securerandom (0.3.1)
     stringio (3.1.1)
+    tailwindcss-ruby (0.1.0-x86_64-linux)
```

Comment:
**issue:** I assume that this change would not work as is.

0.1 version of `tailwindcss-ruby` gem doesn't contain this fix:
https://github.com/flavorjones/tailwindcss-ruby/commit/bd43725db3a328962d04a4c43d9ce4552c45e8c1

Reply:
Thanks for the note. I'm still bootstrapping the gem! Obviously everything will get fixed before a real release. ❤️

Reply:
So looking forward to that!

I want to use it for Jekyll integration as well ;)

Reply:
Yes! I've updated my personal site (jekyll-based) to use this gem and it's totally working for me.

https://github.com/flavorjones/flavorjones.github.io/commit/03b9acea1daeea915a9507c3f81761dc4f1b7dc5

Reply:
@flavorjones amazing work! Thank you

`jekyll-tailwind-cli` gem is just 4 lines of code now thanks to you. I might as well just scrap it. :)